Description: dont include git commit in binary
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 xserver-xorg-video-imx-viv (1-1) unstable; urgency=low
 .
   * Initial release
   * xserver-xorg-video-imx-viv-3.10.17-1.0.1
Author: unknown <unknown>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- xserver-xorg-video-imx-viv-1.orig/EXA/src/makefile.tc
+++ xserver-xorg-video-imx-viv-1/EXA/src/makefile.tc
@@ -52,8 +52,8 @@ prefix ?= /usr
 sysroot ?= /
 
 # get git commit number
-COMMITNR := `git log -n 1 --format=%H`
-DIRTY := `git diff-index --quiet HEAD || echo '-dirty'`
-LOCAL_CFLAGS += -DCOMMIT="${COMMITNR}${DIRTY}"
+#COMMITNR := `git log -n 1 --format=%H`
+#DIRTY := `git diff-index --quiet HEAD || echo '-dirty'`
+#LOCAL_CFLAGS += -DCOMMIT="${COMMITNR}${DIRTY}"
 
 
--- xserver-xorg-video-imx-viv-1.orig/FslExt/src/makefile.tc
+++ xserver-xorg-video-imx-viv-1/FslExt/src/makefile.tc
@@ -52,8 +52,8 @@ prefix ?= /usr
 sysroot ?= /
 
 # get git commit number
-COMMITNR := `git log -n 1 --format=%H`
-DIRTY := `git diff-index --quiet HEAD || echo '-dirty'`
-LOCAL_CFLAGS += -DCOMMIT="${COMMITNR}${DIRTY}"
+#COMMITNR := `git log -n 1 --format=%H`
+#DIRTY := `git diff-index --quiet HEAD || echo '-dirty'`
+#LOCAL_CFLAGS += -DCOMMIT="${COMMITNR}${DIRTY}"
 
 
--- xserver-xorg-video-imx-viv-1.orig/util/autohdmi/makefile.tc
+++ xserver-xorg-video-imx-viv-1/util/autohdmi/makefile.tc
@@ -52,8 +52,8 @@ prefix ?= /usr
 sysroot ?= /
 
 # get git commit number
-COMMITNR := `git log -n 1 --format=%H`
-DIRTY := `git diff-index --quiet HEAD || echo '-dirty'`
-LOCAL_CFLAGS += -DCOMMIT="${COMMITNR}${DIRTY}"
+#COMMITNR := `git log -n 1 --format=%H`
+#DIRTY := `git diff-index --quiet HEAD || echo '-dirty'`
+#LOCAL_CFLAGS += -DCOMMIT="${COMMITNR}${DIRTY}"
 
 
